stdlib/LinearAlgebra/src/generic.jl

```diff
@@ -449,7 +449,7 @@ diag(A::AbstractVector) = throw(ArgumentError("use diagm instead of diag to cons
 # Dot products and norms
 
 # special cases of norm; note that they don't need to handle isempty(x)
-generic_normMinusInf(x) = float(mapreduce(norm, min, x))
+2MinusInf(x) = float(mapreduce(norm, min, x))
```

Comment:
Is this by accident? It's an invalid function name, and it's also not called anywhere.